src/main/java/competition/IdealElectricalContract.java

```diff
@@ -81,38 +75,39 @@ public DeviceInfo turretEncoder() {
         return new DeviceInfo(0, true);
     }
 
-    public DeviceInfo carouselMotor()
-    {
+    public DeviceInfo carouselMotor() {
         return new DeviceInfo(35, false);
     }
 
-    public boolean isCarouselReady(){
+    public boolean isCarouselReady() {
         return true;
     }
 
-    public boolean isCollectorArmLiftingReady() {
+    public boolean isArmReady() {
         return true;
     }
 
-    public DeviceInfo liftingCollectorArmMotor() {
-        return new DeviceInfo(5, true);
+    public DeviceInfo armMotor() {
```

Comment:
sounds like we don't need this armMotor at all anymore? You should be able to remove it